__tests__/app.test.js

```diff
@@ -184,6 +184,31 @@ describe('api/articles', () => {
       expect(articlesArray).toBeSortedBy('created_at', {descending: true})
     })
   })
+  test('GET 200: array is filtered by topic query', () => {
+    return request(app)
+    .get('/api/articles?topic=mitch')
+    .expect(200)
+    .then(({body}) => {
+      const articlesArray = body.articles;
+
+      expect(articlesArray.length).toBe(4)
+
+      articlesArray.forEach(article => {
+        expect(article.topic).toBe('mitch')
+      })
+
+      expect(articlesArray).toBeSortedBy('created_at', {descending: true})
+    })
+  })
+
```

Comment:
Missing Status 200, valid topic query, but has no articles responds with an empty array of articles